component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/GenericInputValidator.java

```diff
@@ -227,7 +227,10 @@ public class GenericInputValidator {
                     new AbstractMap.SimpleImmutableEntry<>(ENVIRONMENT_ID, String.class),
                     new AbstractMap.SimpleImmutableEntry<>(GATEWAY_TYPE, String.class),
                     new AbstractMap.SimpleImmutableEntry<>(PROXY_RESPONSE_CODE, Integer.class),
-                    new AbstractMap.SimpleImmutableEntry<>(TARGET_RESPONSE_CODE, Integer.class))
+                    new AbstractMap.SimpleImmutableEntry<>(TARGET_RESPONSE_CODE, Integer.class),
```

Comment:
when we add these kind of changes, the api manager would break (GA releases unless they populate the same stuff) I guess

Reply:
Ah my bad, since this is choreoFaultSchema, it won't be used by API Manager GA releases.